### PR TITLE
#17_added_&_cofigured_dotenv

### PR DESCRIPTION
**Pull request by IRSHIT033**
## What
  1. Adding and Configuring Dotenv Dependency
  2. added " dotenv " package in package.json
  3. Configured webpack , using DefinePlugin module .

## Outcome
now user can use ".env" file to secure their "Api keys " and confidencial stuffs .

## Screenshots (just for showing added a env variable not added this in "App.js")
Screenshot (48)

### CODE REVIEW

**Comment by IRSHIT033** [comment]
@bhavik2936 can u check I resolved all the merge conflicts

File: webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const path = require('path');
 const webpack = require('webpack');
+const {DefinePlugin}=require('webpack');
+const dotenv=require('dotenv')
```
Comment (bhavik2936) [inline webpack.config.js:4]: Following the conventions, let's have whitespaces and semicolons as used above.

**Review by bhavik2936 — CHANGES_REQUESTED** [review]
Hi Irish, Thanks for your contribution.
As linters and formatter are not configured yet, I have couple of styling suggestions to look over.

**Comment by IRSHIT033** [comment]
ok  I am working on it

**Pull request closed without merging**